xfade.ttl.in

```diff
@@ -65,10 +67,10 @@
 		lv2:maximum 1 ;
 		lv2:portProperty lv2:integer, lv2:enumeration;
 		lv2:scalePoint [
-			rdfs:label "X-fade: Overlapping Cross Fade: [-1..+1]" ;
+			rdfs:label "X-fade: Overlap [-1..+1]" ;
```

Comment:
Shortening the name for the desktop version and everyone else, just because the MOD's default HTML generator cannot handle it is not an option.

The solution here is to create a custom MOD GUI.

Reply:
This makes sense. I re-generated the model from the MOD SDK interface and I got this "trucate" issue. 

If you are not opposed to integrate something generated by this tool, I could try to add new template to MOD SDK to enlarge slightly the generated pedal UI, and makes this reusable to other MOD SDK users. What do you think ?